console/lib/auth_service.js

```diff
@@ -15,14 +15,20 @@
  */
 'use strict';
 
+var path = require('path');
+var fs = require('fs');
 var rhea = require('rhea');
 var Promise = require('bluebird');
+var ca_dir = "/opt/console/authservice-ca";
```

Comment:
I think it would be better to pass this in, rather than hardcode it. Otherwise it makes the code hard to use outside the specific pod.